lib/kaiseki.js

```diff
@@ -10,26 +10,37 @@
 var request = require('request');
 var _ = require('underscore');
 
-var Kaiseki = function(applicationId, restAPIKey, sessionToken) {
+var Kaiseki = function(applicationId, restAPIKey, sessionToken, serverUrl) {
   this.applicationId = applicationId;
   this.restAPIKey = restAPIKey;
   this.masterKey = null;
   this.sessionToken = _.isUndefined(sessionToken) ? null : sessionToken;
+  this.serverUrl = _.isUndefined(serverUrl)? DEFAULT_SERVER_URL : serverUrl;
+  this.mountPath = _.isUndefined(serverUrl)? DEFAULT_MOUNT_PATH : '/parse';
+  //parse open source api server is required revocable session,
+  //origin parse server is optional, after March 25 2015, default is true
+  this.revocable = 1;
 };
 
-Kaiseki.prototype = {
-  API_BASE_URL: 'https://api.parse.com',
+var DEFAULT_MOUNT_PATH = '/1';
```

Comment:
Open-source ParseServer can specified mount path. Parse.com use /1, and the example use /parse